qml/Stage/Stage.qml

```diff
@@ -1726,11 +1726,12 @@ FocusScope {
 
         property int scrollAreaWidth: width / 3
         property bool progressiveScrollingEnabled: false
+        property bool isTouch: false
```

Comment:
Would `wasTouchPress` be a better name or something like that to indicate the last event was a touch event? I'm fine with it as it is though if it just had little comment to state what it represents

Reply:
Jep